Source/WebKit/UIProcess/Cocoa/UIDelegate.mm

```diff
@@ -240,7 +240,7 @@
     if (!delegate)
         return completionHandler(menu);
 
-    auto contextMenuElementInfo = API::ContextMenuElementInfoMac::create(data.webHitTestResultData().value(), data.qrCodePayloadString());
+    auto contextMenuElementInfo = API::ContextMenuElementInfoMac::create(data.webHitTestResultData().value(), data.qrCodePayloadString(), data.hasEntireImage());
```

Comment:
At some point we should make the `ContextMenuElementInfoMac` just take a `const ContextMenuContextData&` parameter.

Not something that needs to be done in this change though.

Reply:
It did do that until recently, when you changed it in https://commits.webkit.org/260897@main. 😆

Reply:
It used to take a `WebHitTestResultData` before, not a `ContextMenuContextData` 😅

Reply:
Strike that, I misremembered what your change did. This is a good idea, since I'll likely need to expose more properties from `ContextMenuElementInfoMac` in the future. I'll do that when I make those additional changes.

Reply:
Actually, since I'm going to make the other change you suggested now, I might as well do that in this patch too.

Reply:
I did this in the commit I just uploaded.